packages/core/database/src/migrations/internal-migrations/5.0.0-01-convert-identifiers-long-than-max-length.ts

```diff
@@ -76,15 +76,10 @@ const renameIndex = async (knex: Knex, db: Database, diff: IndexDiff) => {
     debug(`not renaming index ${full.indexName} because name hasn't changed`);
     return;
   }
-  // link tables are new and do not need to be migrated
+
   // fk indexes can't be easily renamed, and will be recreated by db sync
   // if this misses something due to the loose string matching, it's not critical, it just means index will be rebuilt in db sync
-  if (
```

Comment:
This was ridiculous and nonsensical (I can call it that because it was my own code) and came from a misunderstanding of the localizations_links tables and a bug that has since been fixed